torch_geometric/nn/aggr/multi.py

```diff
@@ -2,23 +2,51 @@
 
 import torch
 from torch import Tensor
+from torch.nn import Linear
 
 from torch_geometric.nn.aggr import Aggregation
 from torch_geometric.nn.resolver import aggregation_resolver
 
 
 class MultiAggregation(Aggregation):
-    def __init__(self, aggrs: List[Union[Aggregation, str]],
-                 aggrs_kwargs: Optional[List[Dict[str, Any]]] = None):
+    r"""Performs aggregations with one or more aggregators and combines and
+        aggregated results.
+
+    Args:
+        aggrs (list): The list of aggregation schemes to use.
+        aggrs_kwargs (list, optional): Arguments passed to the
+            respective aggregation function in case it gets automatically
+            resolved. (default: :obj:`None`)
+        combine_mode (string or Aggregation, optional): The combine mode
```

Comment:
Does it make sense to support `Aggregation` here? I think for these dense aggregations, we should rely on dense implementation (e.g., torch.mean).

Reply:
Updated.